docs/reference/metadata.md

```diff
@@ -214,15 +254,57 @@ Indentation options control aspects of the auto-indentation mechanism.
 
 `unIndentedLinePattern`
 : *Regex.*
+  If the `increaseIndentPattern` and/or `decreaseIndentPattern` *Regex.* also matches on the current line.
   The auto-indenter will ignore
-  lines matching this regex
-  when computing the next line's indentation level.
+  the effects made by `increaseIndentPattern` and/or `decreaseIndentPattern`
+  when computing the indentation level of lines after current line.
+
+  If in `.tmPreferences`:
 
   ```xml
+  <key>increaseIndentPattern</key>
+  <string><![CDATA[^\s*[A-z]+]]></string>
   <key>unIndentedLinePattern</key>
-  <string>insert regex here</string>
+  <string><![CDATA[^\s*unindented$]]></string>
```

Comment:
I guess for the examples below to work, the `$` anchor would have to be removed?